.github/workflows/generate-docs.yml

```diff
@@ -17,15 +17,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          token: ${{ secrets.ZEEK_BOT_TOKEN }}
+          submodules: "recursive"
 
-      - name: Sync Submodules
-        run: |
-          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
-          git submodule sync --recursive
-          git -c "http.extraheader=$auth_header" -c protocol.version=2 \
-            submodule update --init --force --recursive --depth=1
-          ( cd doc && git checkout master )
```

Comment:
Was this authheader stuff doing anything that would still matter?

Reply:
I'm not sure, but I noticed one thing: when you use the `checkout` action to do the recursing, it internally relies on that extraheader too:

https://github.com/actions/checkout/blob/main/src/git-auth-helper.ts#L55

So I think that this approach was needed in the past and is today offered directly by the `checkout` action. (And, fwiw, we do use `checkout` with its ability to recurse elsewhere already.)